content/zh/docs/concepts/configuration/organize-cluster-access-kubeconfig.md

```diff
@@ -280,6 +281,53 @@ kubeconfig 文件中的文件和路径引用是相对于 kubeconfig 文件的位
 命令行上的文件引用是相对于当前工作目录的。
 在 `$HOME/.kube/config` 中，相对路径按相对路径存储，绝对路径按绝对路径存储。
 
+<!--
+## Proxy
+
+You can configure `kubectl` to use proxy by setting `proxy-url` in the kubeconfig file, like:
+
+```yaml
+apiVersion: v1
+kind: Config
+
+proxy-url: https://proxy.host:3128
+
+clusters:
+- cluster:
+  name: development
+
+users:
+- name: developer
+
+contexts:
+- context:
+  name: development
+
+```
```

Comment:
This yaml is not translated/modified during localization, so you can safely leave it as is.
The translation is only about line 285-287.

Reply:
Got it, thanks.